IbisPlugins/CameraCalibration/cameracalibrationplugininterface.cpp

```diff
@@ -97,6 +97,7 @@ bool CameraCalibrationPluginInterface::WidgetAboutToClose()
         m_cameraCalibrationWidget->close();
 
     ClearCameraViews();
+    m_cameraCalibrator->ClearCalibrationData();
```

Comment:
I agree we failed to clear the data before and that was a problem, but it should be done in the destructor to ensure that if you deactivate and activate the plugin, the data is still there.